agave_app/qtControls/controlFactory.h

```diff
@@ -0,0 +1,31 @@
+#pragma once
+
+#include "renderlib/uiInfo.hpp"
+#include "renderlib/core/prty/prtyProperty.h"
+
+#include <memory>
+
+class QNumericSlider;
+class QCheckBox;
+class QComboBox;
+
+QNumericSlider*
+create(const FloatSliderSpinnerUiInfo* info, std::shared_ptr<prtyProperty<float>> prop);
+
+QNumericSlider*
+create(const IntSliderSpinnerUiInfo* info, std::shared_ptr<prtyProperty<int>> prop);
```

Comment:
Missing equivalent declaration for `addRow`?

Reply:
added...